docs/sources/write/style-guide/word-list/index.md

```diff
@@ -25,6 +25,19 @@ This page is a work in progress.
 
 ## A
 
+<!-- vale Grafana.Agentless = NO -->
+
+### agentless
+
+Grafana Alloy is a vendor-neutral distribution of the OpenTelemetry (OTel) Collector.
+It's not an agent.
+
+If you're talking about why and how to send signals directly from app to prefer no-collector to agentless.
```

Comment:
```suggestion
If you're talking about why and how to send signals directly from app to prefer no collector to agentless.
```

Should this be `no collector` or `no-collector`. The OTel docs describe it without the hyphen.

Reply:
Updated in https://github.com/grafana/writers-toolkit/pull/741 to be `no-collector`